sdcm/tester.py

```diff
@@ -441,9 +441,10 @@ def _init_test_timeout_thread(self) -> threading.Timer:
         self.log.info('Test start time %s, duration is %s and timeout set to %s',
                       start_time, self.test_duration, end_time)
         self.argus_update_status(TestStatus.RUNNING)
+        timeout_event = TestTimeoutEvent(start_time=self.start_time, duration=self.test_duration)
```

Comment:
Is there any effect to create the TestTimeoutEvent very early?
How about passing the self.start_time, and self.test_duration as arguments for calling kill_the_test?

Reply:
fixed